templates/client-daemonset.yaml

```diff
@@ -129,6 +131,10 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: spec.nodeName
+            - name: HOST_IP
+              valueFrom:
+                fieldRef:
+                  fieldPath: status.hostIP
```

Comment:
can you also set `CONSUL_HTTP_SSL_VERIFY=false` if autoencrypt is used since the ca cert doesn't exist.

Reply:
Yeah, good idea. At first, I thought it's a bad idea because skipping SSL verification is always a bad idea, but since it's for local traffic, we're not violating any security assumptions.